### ISSUE TITLE
adding linux command-line running

### ISSUE THREAD

**Issue by jan-perl** [ObjectVision/GeoDMS#138]
The wiki describes how to use the GUI under linux, but not how to run the command-line version.

https://github.com/ObjectVision/GeoDMS/wiki/GeoDMSOnLinux

Command-line linux enables running pre-defined configurations, possibly timed too, under linux.

**Comment by MaartenHilferink** [comment]
See also #16

**Comment by eoudejans** [comment]
Confirming that GeoDmsRun.exe can be run using wine, at least for operator.dms and regression.dms in gdal_storage in in our  [GeoDms_Test](https://github.com/ObjectVision/GeoDMS-Test) repository.

**Comment by eoudejans** [comment]
I have updated the documentation for the [GeoDMSOnLinux](https://github.com/ObjectVision/GeoDMS/wiki/GeoDMSOnLinux) which inludes the latest state of running GeoDmsRun.exe on Fedora Linux 37.

**Comment by eoudejans** [comment]
Example output of reading and writing several files with gdal using `wine GeoDmsRun.exe`
![image](https://github.com/ObjectVision/GeoDMS/assets/96182097/0dbd81c0-e2d5-40aa-8053-965200a33075)

**Comment by eoudejans** [comment]
Exploration is currently being conducted as how to best compile GeoDMS natively for linux.

**Comment by eoudejans** [comment]
I have recently updated the GeoDMS on Linux docs, which at the moment consists of my experience of running GeoDMSRun.exe on Fedora Linux, see: [https://github.com/ObjectVision/GeoDMS/wiki/GeoDMS-on-Linux](https://github.com/ObjectVision/GeoDMS/wiki/GeoDMS-on-Linux). 

That being said, we have at the moment mixed experience with using Wine. On Windows Subsystem Linux: Ubuntu we were not able to run GeoDmsRun using Wine. We also know that some larger Linux cluster computers are not able to use Wine to run GeoDMSRun. 

Closing issue, as running the command line version is updated [here](https://github.com/ObjectVision/GeoDMS/wiki/GeoDMS-on-Linux).
